net-fs/curlftpfs/curlftpfs-0.9.2.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="Filesystem for accessing FTP hosts based on FUSE and libcurl"
+DESCRIPTION="Usage. First start the userland_fs server:
+/system/servers/userlandfs_server curlftpfs &
+
+Then mount an ftp share:
+
+mount -t userlandfs -p 'curlftpfs -o option1,option2 \\
+    ftp://someserver.somewhere/some/directory' /local/mount/directory
+
+Possible options are listed in the man page (includes SSL, proxy etc support)."
+HOMEPAGE="http://curlftpfs.sourceforge.net/"
+COPYRIGHT="Robson Braga Araujo <braga (a) users sf net>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/curlftpfs/curlftpfs-$portVersion.tar.gz"
+CHECKSUM_SHA256="4eb44739c7078ba0edde177bdd266c4cfb7c621075f47f64c85a06b12b3c6958"
+PATCHES="curlftpfs-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+        curlftpfs$secondaryArchSuffix = $portVersion
+	addon:curlftpfs$secondaryArchSuffix
+        cmd:curlftpfs$secondaryArchSuffix
+        "
+REQUIRES="
+        haiku$secondaryArchSuffix
+        glib2$secondaryArchSuffix >= 2
+        lib:libcurl$secondaryArchSuffix
+        gettext$secondaryArchSuffix
+        gettext${secondaryArchSuffix}_libintl
```

Comment:
Thanks @korli for your feedback.
I knew my initial changes were not enough so I was going to ask for feedback... and there it is, before I even start asking ;-)
So that's cool, because now I just need to apply your changes!